src/main/java/ua/bookstore/online/exception/CustomGlobalExceptionHandler.java

```diff
@@ -56,6 +58,11 @@ protected ResponseEntity<ErrorResponseDto> handleMethodNotFound(EntityNotFoundEx
         return getResponseEntity(NOT_FOUND, ex.getMessage());
     }
 
+    @ExceptionHandler(AccessDeniedException.class)
+    protected ResponseEntity<ErrorResponseDto> handleMethodNotFound(AccessDeniedException ex) {
+        return getResponseEntity(FORBIDDEN, ex.getMessage());
+    }
+
     @ExceptionHandler({SpecificationProviderNotFoundException.class})
     protected ResponseEntity<ErrorResponseDto> handleMethodSpecificationProviderNotFound(
             SpecificationProviderNotFoundException ex) {
```

Comment:
The same

Reply:
I leave. For me, it is normal name for this method.